src/app/Main.java

```diff
@@ -60,7 +61,7 @@ public static void main(String[] args) {
         FileStationDataAccessObject stationDataAccessObject;
         try {
             stationDataAccessObject = new FileStationDataAccessObject("./revisedStopData.txt", new StationFactory(),
-                    new TrainFactory(), new GOStationApiClass(), new GOVehicleApiClass());
+                    new TrainFactory(), new GOStationApiClass(), new GOVehicleApiClass(), new GOStationCoordinatesApiClass());
```

Comment:
Wouldn't it be redundant since GOStationCOordinatesApiClass isn't used anywhere and GOVehicleAPiClass already has latitude and longitude retrieval?

Reply:
GOStationCoordinatesApiClass is the station coordinates not vehicles, both api classes are needed for lat1 lon1 / lat2 lon2

Reply:
My bad, didn't see it's for station!! 

For retrieving station coordinates, would it be possible to build on GOStationApiClass() instead of creating a new Api class? GOStationApiClass()'s api call endpoint (Stop/Details/{StopCode}) also includes station postition.
<img width="344" alt="image" src="https://github.com/JasonBarahan/TrackMyTransit/assets/123687946/f80d8fea-d435-4c8e-8e07-f626c60a212a">